app/src/gecko/java/org/mozilla/focus/iwebview/WebViewProvider.java

```diff
@@ -97,6 +97,14 @@ public void onResume() {
 
         }
 
+        @Override
+        public void onStart() {
```

Comment:
Why onStart/onStop, rather than onCreate/onDestroy? (is onDestroy not guaranteed to be called?)

Reply:
`onDestroy` isn't guaranteed to be called. Generally, it will be called when the app will continue (e.g. Activity or Fragment is being explicitly removed) but it's safer to call in onStop.

Then again, with my mindset, nothing would go in onDestroy so maybe I need to rethink that. 🤔